spec/javascripts/itsi_editor_spec.coffee

```diff
@@ -144,6 +145,27 @@ describe "ITSI Editor", () ->
 
       expect(children.style.display).toBe("block")
 
+    it "toggles does not children visibility when not confirmed", ->
+      props =
+        title: "test title"
+        toHide: "hidden"
+        confirmHide: -> false
+        data:
+          is_hidden: false
+          update_url: "fake_update_url"
+      sectionEditorElement = jasmine.react.renderComponent "itsi_authoring/section_editor_element", props
+
+      checkbox = jasmine.react.findTag sectionEditorElement, "input"
+      children = (jasmine.react.findClass sectionEditorElement, "ia-section-editor-elements").getDOMNode()
+
+      expect(children.style.display).toBe("block")
+
+      request = jasmine.react.captureRequest ->
+        jasmine.react.change checkbox, {"target": {"checked": false}}
+      expect(request).toBe(undefined)
+
+      expect(children.style.display).toBe("block")
```

Comment:
We might also assert that when alert is confirmed/dismissed, that the block is hidden right?  I am not sure if there are good examples for how to do this.  I found this SO discussion http://stackoverflow.com/questions/19519328/testing-javascript-alerts-with-jasmine  Its probably fine to punt on this though.